### PR TITLE
Added some small things

### PR DESCRIPTION
**Pull request by swhahn**
Hello,
nice package. I added some small things.
With kind regards,
Stefan

### CODE REVIEW

File: counsel-gtags.el [hunk 1]
```diff
@@ -245,8 +248,8 @@ Always update if value of this variable is nil."
          (with-temp-buffer
            (let* ((options (cl-case counsel-gtags-path-style
                              (absolute "-Poa")
-                             (root "-Poc")
-                             (relative ""))))
+                             (root "-Poa")
```
Comment (syohex) [inline counsel-gtags.el:251]: Why did you change this option ? `-Poa` shows absolute path, not path from root directory.

Comment (swhahn) [reply counsel-gtags.el:251]: Sorry, running global 5.7.1 does not have option c; so I changed it to a.

**Comment by syohex** [comment]
I have cherry-picked except https://github.com/syohex/emacs-counsel-gtags/pull/10/commits/cfc860e6aecb82f43343170596e4ec056bf32b3d

File: counsel-gtags.el (hunk 1 above)
Comment (syohex) [reply counsel-gtags.el:251]: I recommend you to use GNU Global 6.2.3 or higher version same as helm-gtags.

**Pull request closed without merging**